archetype-models/archetype-descriptor/src/main/mdo/archetype-descriptor.mdo

```diff
@@ -162,6 +162,13 @@
           <required>false</required>
           <description>Encoding to use when filtering content.</description>
         </field>
+        <field xml.attribute="true">
+          <name>includeCondition</name>
+          <version>1.0.0+</version>
```

Comment:
I think about put 1.1.0+ here ... we also need update version for model in plugin configuration

Reply:
It should be backwards compatible so that's why I put it in v1.0.0, but I'm fine with both. I'm not sure what the implications would be when adding it for 1.0.0

Reply:
we will publish next schema version, for backward compatible users can use old schema without new field ....